solutions/devsprint-pedro-menezes-1/DeliveryAppChallenge/Modules/Product/Scenes/TatianaRico/TRLoginViewController.swift

```diff
@@ -56,53 +52,42 @@ class TRLoginViewController: UIViewController {
         return .lightContent
     }
     
-    func verifyLogin() {
+    private   func verifyLogin() {
         if let _ = UserDefaultsManager.UserInfos.shared.readSesion() {
-            coordinator.verifyLogin()
+            coordinator.goToHomeViewController()
         }
     }
     
+    private func placeholderTextFieldInicial() {
+#if DEBUG
+        emailTextField.text = "clean.code@devpass.com"
+        passwordTextField.text = "111111"
+#endif
+    }
+    
     func alertConection(titleAlert: String, messageAlert: String, actionMsgAlert: String) {
```

Comment:
funçao começa sempre com verbo
nunca abrevia nomes (principio de pronuncia de nomenclatura)
to vendo um `Msg`ali 👀